portfolio/src/main/webapp/maps.js

```diff
@@ -0,0 +1,59 @@
+const mapsKey = config.MAPS_KEY;
+const surfSpots = ['pv', 'tpz', 'bw', 'lf', 'nmbp', 'porto'];
+window.onload = createScriptTagForMap();
+
+
+function createMap() {
+    const map = new google.maps.Map(
+        document.getElementById('map-container'),
+        {center: {lat:33.835, lng: -118.390}, zoom: 12});
```

Comment:
Could you add a comment describing the default map center? E.g. "Default to southern California" or wherever.